package.json

```diff
@@ -191,24 +191,23 @@
     "@types/parse-gitignore": "1.0.2",
     "@types/prettyjson": "0.0.33",
     "@types/semver": "7.5.8",
-    "@types/uuid": "9.0.8",
     "@types/ws": "8.5.14",
     "@vitest/coverage-v8": "1.6.0",
-    "c8": "9.1.0",
+    "c8": "10.1.3",
     "cheerio": "1.0.0",
     "eslint-plugin-sort-destructure-keys": "2.0.0",
     "eslint-plugin-workspace": "file:./tools/lint-rules",
     "form-data": "4.0.1",
-    "fs-extra": "11.2.0",
-    "husky": "8.0.3",
-    "is-ci": "3.0.1",
+    "fs-extra": "11.3.0",
+    "husky": "9.1.7",
```

Comment:
requires node 18 (so do we), technically no other breaking changes but I updated to the new format: https://github.com/typicode/husky/releases/tag/v9.0.1